packages/dds/tree/src/test/simple-tree/treeContext.spec.ts

```diff
@@ -115,8 +115,7 @@ describe("TreeBranch", () => {
 			assert.deepEqual([...branchBranch.root], ["y"]);
 			assert.throws(
 				() => view.rebaseOnto(branch),
-				(e: Error) =>
-					validateAssertionError(e, /The main branch cannot be rebased onto another branch./),
+				(e: Error) => validateAssertionError(e, /cannot be rebased onto another branch./),
```

Comment:
I had to relax this test, since the assert message in the inline comment and short code table were in disagreement.

A follow-up should decide which message is correct, fix both and adjust this test again.

@noencke